docs/reference/troubleshooting.asciidoc

```diff
@@ -50,6 +50,8 @@ include::troubleshooting/data/add-tier.asciidoc[]
 
 include::troubleshooting/data/diagnose-unassigned-shards.asciidoc[]
 
+include::modules/discovery/discovery.asciidoc#troubleshooting-discovery[]
```

Comment:
This `include` statement won't work because it's referencing the HTML header. These type of statements are tricky, especially when you're only trying to pull in part of a page and not the entire page. I'll push a fix for this.